packages/client/mutations/handlers/handleRemoveReflectTemplatePrompt.ts

```diff
@@ -1,10 +1,21 @@
 import {RecordSourceSelectorProxy} from 'relay-runtime'
 import pluralizeHandler from './pluralizeHandler'
-import removeFromRefs from '../../utils/relay/removeFromRefs'
+import {IRetrospectiveMeetingSettings, MeetingTypeEnum} from '~/types/graphql'
+import safeRemoveNodeFromArray from '~/utils/relay/safeRemoveNodeFromArray'
 
-const handleRemoveReflectTemplatePrompt = (promptId: string, store: RecordSourceSelectorProxy) => {
-  store.delete(promptId)
-  removeFromRefs(promptId, store, {ReflectTemplate: ['prompts']})
```

Comment:
`removeFromRefs` was only being used in this file. After removing it here, I've deleted the file